akka-docs/src/main/paradox/remoting.md

```diff
@@ -2,10 +2,13 @@
 
 For an introduction of remoting capabilities of Akka please see @ref:[Location Transparency](general/remoting.md).
 
+We recommend that you prefer @ref:[Akka Cluster](cluster-usage.md) over using remoting directly. As remoting is the
+underlying module that allows for Cluster, it is still useful to understand details about it though.
```

Comment:
Snuck this extra caution in here

Reply:
"recommend that you prefer" is strange wording

I think "We recommend Akka Cluster over" would be enough